hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Yeah, there are two reasons why I dislike `local` variables. First is it confuses readers (mostly because that is rarely used feature in the wild AFAIK) and the second is code portability (but that is not our issue here).